coreclr-debug/NuGet.config

```diff
@@ -4,7 +4,5 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <!-- This dependency can be commented out once we push packages to nuget for a real release -->
-    <add key="coreclrdebug" value="https://www.myget.org/F/coreclr-debug/api/v3/index.json" />    
```

Comment:
do we want to comment this out instead of delete it? Are we still planning on pushing daily builds to myget?